### ISSUE TITLE
Tests: Block User test requires enterprise license to work

### ISSUE THREAD

**Issue by sagor999** [gitpod-io/gitpod#7800]
### Bug description

When running integration tests, block user test fails with error:
`blockuser_test.go:71: cannot perform AdminBlockUser: "jsonrpc2: code 555 message: enterprise license required"`

### Steps to reproduce

Run integration test on new gitpod deployment

### Expected behavior

Test should pass even if enterprise license is missing

**Comment by kylos101** [comment]
Hi @geropl I am currently planning [an epic for integration tests for team workspace](https://github.com/gitpod-io/gitpod/issues/8799), and bumped into this issue [given this component tagging](https://github.com/gitpod-io/gitpod/blob/50c2cf59f1898df599cf1a436af5ff4716f95194/test/tests/components/server/blockuser_test.go#L21-L24). In hindsight, it seems like it belongs on the webapp side of the house?

**Comment by geropl** [comment]
@kylos101 Indeed. Is it possible for you to exclude that tests/label it as "team webapp" for now? If yes, I'd love to close this issue for now, and tackle the underlying problem once we find resources to dig into testing again.

**Comment by kylos101** [comment]
Yes, not a problem. [This script](https://github.com/gitpod-io/gitpod/blob/7ac5d9d3150dd6fb72f32915afd1923574b3811b/test/leeway-build.sh#L1) creates test binaries for each component, and we will [modify the entry point](https://github.com/gitpod-io/gitpod/blob/7ac5d9d3150dd6fb72f32915afd1923574b3811b/test/entrypoint.sh). For example, to allow us to specify different component names, this way, when we call it from our werft job (which'll be for team workspace) it'll just certain components.

**Comment by kylos101** [comment]
Closing as per this [comment](https://github.com/gitpod-io/gitpod/issues/7800#issuecomment-1066971852).
